package.json

```diff
@@ -3,20 +3,23 @@
   "version": "0.1.7",
   "description": "A set of fully customizable React components made with <3 with styled-components",
   "main": "dist/index.js",
+  "module": "dist/esm/index.js",
   "scripts": {
-    "build": "webpack --config webpack/native.config.babel.js && webpack --config webpack/web.config.babel.js",
+    "build": "yarn run build:dev && yarn run build:prod",
+    "build:prod": "cross-env NODE_ENV=production rollup -c",
+    "build:dev": "cross-env NODE_ENV=development rollup -c",
     "build:documentation": "styleguidist build",
     "dev": "yarn run start:styleguidist",
-    "gen": "babel-node --presets babel-preset-env ./bin/generateComponent",
+    "gen": "babel-node --presets @babel/preset-env ./bin/generateComponent",
```

Comment:
Do we really need the preset here since we already have it in the .babelrc file ?

Reply:
Yes we do otherwise, es6 is not understood